kong/db/dao/plugins/go.lua

```diff
@@ -36,6 +36,47 @@ do
   end
 end
 
+do
+  local pluginserver_proc
+
+  function go.manage_pluginserver()
+    if pluginserver_proc then
+      -- already being managed
+      return
+    end
+
+    if ngx.worker.id() ~= 0 then
+      -- only one manager
+      pluginserver_proc = true
+      return
+    end
+
+    ngx_timer_at(0, function ()
```

Comment:
```lua
ngx_timer_at(0, function(premature)
  if premature then
    return
  end
...
```

is better, to avoid prolonging the exit of worker process by the timer.

Also in Kong the styling does not leaves a space between `function` and `()`.